dist-scripts/lib/notificationUtils.js

```diff
@@ -1,37 +1,45 @@
+import { v4 as uuidv4 } from 'uuid';
 export async function createNotification(db, recipientWalletAddress, type, title, message, ctaUrl, relatedQuestId, relatedQuestTitle, relatedSquadId, relatedSquadName, relatedUserId, relatedUserName, relatedInvitationId, rewardAmount, rewardCurrency, badgeId) {
+    var _a;
```

Comment:
nit: consider using a descriptive name so we can know the intent or add some inline comment